### PR TITLE
Html forms

### PR DESCRIPTION
**Pull request by DirCd**
Screenshot_1

### CODE REVIEW

File: Forms.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        <label for="cfile">Upload a Profile Picture:</label>
+        <input type="file" id="cfile" name="cfile" accept=".png,.jpg"><br>
+        <label for="Url">Website URL:</label>
+        <input type="url" id="Url" name="Url"><br>
+        <label for="exl" >Experience level with HTML:</label>
+        <input type="range" id="exl" name="exl" min="1" max="10">
+        <output id="out">5</output>/10
+    </fieldset>
+    <input type="submit" value="Submit">
+    <input type="reset">
```
Comment (IvanCeban) [inline Forms.html:82]: reset input should have a value as well

File: Forms.html [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    <fieldset>
+        <legend>Additional Details:</legend>
+        <label for="hdha">How did you hear about us?</label>
+        <select id="hdha" name="hdha">
+            <option value="internet">Internet</option>
+            <option value="friend">Friend</option>
+            <option value="adv">Advertisement</option>
+            <option value="magazine">Magazine</option>
+        </select><Br>
+        <label for="f-c">Feedback/Comments</label>
```
Comment (IvanCeban) [inline Forms.html:67]: avoid using such id and names - "f-c". It is clear for you now in this part of code, but if you start using the id or name in an external CSS or JS file you will get lost. 
Always use meaningfully names, classes, ids

File: Forms.html [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        <label for="color">Favorite color:</label>
+        <input type="color" id="color" name="color">
+    </fieldset>
+    <fieldset>
+        <legend>Additional Details:</legend>
+        <label for="hdha">How did you hear about us?</label>
+        <select id="hdha" name="hdha">
+            <option value="internet">Internet</option>
+            <option value="friend">Friend</option>
+            <option value="adv">Advertisement</option>
```
Comment (IvanCeban) [inline Forms.html:64]: c'mon, don't be lazy.
Write the word in full. Imagine that on backed will work somebody else and he doesn't understand what you mean by "adv"

File: Forms.html [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+        <input type="password" id="cpwd" name="cpwd" required><br>
+        <label >Gender:</label>
+        <input type="radio" id="male" name="Gen" value="Male">
+        <label for="male">Male</label>
+        <input type="radio" id="female" name="Gen" value="Female">
+        <label for="female">Female</label>
+        <input type="radio" id="other" name="Gen" value="Other">
+        <label for="other">Other</label><br>
+        <label for="DOB">Date of Birth:</label>
+        <input type="date" id="DOB" name="DOB">
```
Comment (IvanCeban) [inline Forms.html:27]: no uppercase for id's.
Only lowercase or kebab-case / snake_case

File: Forms.html (hunk 3 above)
Comment (IvanCeban) [reply Forms.html:64]: I mean not only this one. But all thee abbreviations you used in the rest of elements

**Review by IvanCeban — CHANGES_REQUESTED** [review]

File: Forms.html (hunk 1 above)
Comment (DirCd) [reply Forms.html:82]: image

File: Forms.html (hunk 2 above)
Comment (DirCd) [reply Forms.html:67]: image

**Pull request closed without merging**